R/scripts/create_seeding.R

```diff
@@ -38,61 +38,126 @@ library(tidyr)
 library(purrr)
 
 option_list = list(
-  optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("CONFIG_PATH"), type='character', help="path to the config file")
+  optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("CONFIG_PATH"), type='character', help="path to the config file"),
+  optparse::make_option(c("-s", "--source"), action="store", default="CSSE", type='character', help="source of case data: USAFacts or CSSE"),
+  optparse::make_option(c("-d", "--data"), action="store", default=file.path("data","case_data","case_data.csv"), type='character', help="path to the case data file")
 )
 
-opts = optparse::parse_args(optparse::OptionParser(option_list=option_list))
+opt = optparse::parse_args(optparse::OptionParser(option_list=option_list))
 
-config <- covidcommon::load_config(opts$c)
+config <- covidcommon::load_config(opt$c)
 if (length(config) == 0) {
   stop("no configuration found -- please set CONFIG_PATH environment variable or use the -c command flag")
 }
 
-jhucsse <- covidImportation::get_clean_JHUCSSE_data(aggr_level = "UID", 
-                                   last_date = as.POSIXct(lubridate::ymd(config$end_date)),
-                                   case_data_dir = file.path('importation',config$spatial_setup$setup_name,"case_data"),
-                                   save_raw_data=TRUE,
-                                   us_data_only=FALSE)
-
-print("Successfully pulled JHU CSSE data for seeding.")
-
 all_times <- lubridate::ymd(config$start_date) +
   seq_len(lubridate::ymd(config$end_date) - lubridate::ymd(config$start_date))
 
-geodata <- report.generation:::load_geodata_file(file.path(config$spatial_setup$base_path, config$spatial_setup$geodata),5,'0',TRUE)
 
-all_geoids <- geodata[[config$spatial_setup$nodenames]]
 
-incident_cases <- jhucsse %>%
-  dplyr::filter(FIPS %in% all_geoids) %>%
-  dplyr::select(Update, FIPS, incidI)
 
-incident_cases$Update <- as.Date(incident_cases$Update)
+# Get the data
+
+
+
+# get data if a US model  ---------------------------------
+
+if (is.null(config$spatial_setup$us_model) || config$spatial_setup$us_model==TRUE){
```

Comment:
This will throw a weird error if spatial_setup isn't part of the config.  I think since we're aiming to push to master, we want to fix that here.

Reply:
Are you sure? when i test it it's fine. If it's missing, it returns a NULL.

Reply:
oh, i guess this was an old bug.  i'm not up to speed.  Yeah, no need to change this